guides/common/modules/proc_synchronizing-templates-with-an-existing-repository.adoc

```diff
@@ -45,7 +45,14 @@ Do not specify a passphrase.
 ----
 # sudo --user foreman ssh _git.example.com_
 ----
-. Configure the Template Sync plugin settings on a *Template Sync* tab.
-.. Change the *Branch* setting to match the target branch on a Git server.
-.. Change the *Repo* setting to match the Git repository.
-For example, for the repository located in `git@git.example.com/templates.git` set the setting into `git@git.example.com/templates.git`.
+. In the {ProjectWebUI}, provide details for your Git repository:
+.. Navigate to *Administer > Settings*.
+.. Click the *Template Sync* tab.
+.. Change the *Branch* setting to define the target branch of the repository.
+.. Change the *Repo* setting to define the repository URL.
+.. If you want to use an HTTP proxy to connect to the repository, change the *HTTP proxy policy* setting to define your HTTP proxy policy.
```

Comment:
There are many settings on the template sync tab that that affect the template sync (or rather eliminate the need to input them manually while syncing). Maybe you could mention that it is possible to configure these settings too as an optional step, possibly with the link to template sync settings. But again, this is more stylistic than technical suggestion so apply or disregard at your own discretion.

Reply:
At this point in the procedure, users will see the settings along with descriptions of what the settings do. I don't think that we need to explicitly tell them that those settings are there at a moment when they can see them for themselves. We do mention the settings in the 'next steps' procedures (importing/exporting templates) . These procedures actually refer back to the Template Sync tab. I think that's enough.

So, no change.